templates/services/jobs/scheduled/manifest.yml

```diff
@@ -0,0 +1,38 @@
+# The manifest for the "{{.Name}}" job.
+# Read the full specification for the "{{.Type}}" type at:
+#  https://github.com/aws/copilot-cli/wiki/Manifests#scheduled-job
+
+# Your job name will be used in naming your resources like log groups, ECS Tasks, etc.
+name: {{.Name}}
+# The "architecture" of the job you're running.
+type: {{.Type}}
+
+image:
+  # Docker build arguments. You can specify additional overrides here. Supported: dockerfile, context, args
+  build: {{.Image.Build.BuildArgs.Dockerfile}}
+
+# Number of CPU units for the task.
+cpu: {{.CPU}}
+# Amount of memory in MiB used by the task.
+memory: {{.Memory}}
+
+# The number of times to retry the job before failing.
+retries: 1
+# The timeout after which to stop the job if it's still running. 
+timeout: 900
+# The trigger for your job. You can set the `rate` or `cron` and `eventpattern` fields.  
+trigger:
+  rate: every 20 minutes
+
+# Optional fields for more advanced use-cases.
+#
+#variables:                    # Pass environment variables as key value pairs.
+#  LOG_LEVEL: info
+#
+#secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store.
+#  GITHUB_TOKEN: GITHUB_TOKEN  # The key is the name of the environment variable, the value is the name of the SSM parameter.
+
+# You can override any of the values defined above by environment.
+#environments:
+#  test:
+#    count: 2               # Number of tasks to run for the "test" environment.
```

Comment:
There is no `count` field in the general section above, do we want to update this?

Reply:
fixed.